src/main/java/marquez/db/models/DatasetRow.java

```diff
@@ -15,30 +15,20 @@
 package marquez.db.models;
 
 import java.time.Instant;
-import java.util.Optional;
 import java.util.UUID;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.NonNull;
-import lombok.RequiredArgsConstructor;
-import lombok.ToString;
+import lombok.Data;
 
-@RequiredArgsConstructor
-@EqualsAndHashCode
-@ToString
+@Data
```

Comment:
Should we use `@Value` instead. It's like `@Data`, but I think it also gives us immutability built in.

Reply:
We'll also need setters and `@Data` generates mutators for us while `@Value` only provides `@AllArgsConstructor` to initialize fields.